services/core-web/src/components/modalContent/ExplosivesPermitViewModal.tsx

```diff
@@ -157,7 +157,7 @@ export const ExplosivesPermitViewModal: FC<ExplosivesPermitViewModalProps> = (pr
         ...permit,
         issue_date: permit.issue_date,
         expiry_date: permit.expiry_date,
-        is_closed: permit.is_closed ? "Closed" : "Open",
```

Comment:
I think instead of this, the other one (line 167) needs to be changed to `is_closed`.  Based on the columns at line 113ish, the table is looking for that property.

Reply:
Updated.